### ISSUE TITLE
【Docathon】.. py:class:: 

### ISSUE THREAD

**Issue by Echo-Nie** [PaddlePaddle/docs#7147]
### 文档链接&描述 Document Links & Description

https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/BatchNorm1D_cn.html
![Image](https://github.com/user-attachments/assets/07d11b5b-8144-4a9f-bdd7-41197a621c9d)

There are many documents with an extra "class" in the introduction, and I looked at what was written in the source code. `paddle.nn.BatchNorm1D(num_features, momentum=0.9, epsilon=1e-05, weight_attr=None, bias_attr=None, data_format='NCL', use_global_stats=None, name=None)`. Spaces should be added here, as follows:
```
.. py:class:: paddle.nn.BatchNorm1D(num_features, momentum=0.9, epsilon=1e-05, weight_attr=None, bias_attr=None, data_format='NCL', use_global_stats=None, name=None)
```

### 请提出你的建议/提个PR修复这个问题 Please give your suggestion/update a PR to fix it

Add space
